config/proposer/loader/loader.go

```diff
@@ -329,6 +329,17 @@ func reviewGasLimit(gasLimit validator.Uint64) validator.Uint64 {
 	if gasLimit == 0 {
 		return validator.Uint64(params.BeaconConfig().DefaultBuilderGasLimit)
 	}
-	// TODO(10810): add in warning for ranges
+
+	// Warning for ranges that might be problematic
+	defaultGasLimit := params.BeaconConfig().DefaultBuilderGasLimit
+	// If gas limit is very low (below 10% of default), warn about potential issues
+	if gasLimit < validator.Uint64(defaultGasLimit/10) {
```

Comment:
```suggestion
	if gasLimit <= validator.Uint64(defaultGasLimit/10) {
```

If you use `<=` then it would catch a single truncated 0 on the value.